<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="layoutHolder">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QTabWidget" name="twNavigation">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tabWeek">
         <attribute name="title">
          <string>Week</string>
         </attribute>
         <layout class="QGridLayout" name="layoutWeek">
          <item row="1" column="0">
           <widget class="HeaderTableItem" name="dayMonday" native="true"/>
          </item>
          <item row="1" column="2">
           <widget class="HeaderTableItem" name="dayWednesday" native="true"/>
          </item>
          <item row="2" column="3">
           <widget class="HeaderTableItem" name="dayAny" native="true"/>
          </item>
          <item row="2" column="1">
           <widget class="HeaderTableItem" name="daySaturday" native="true"/>
          </item>
          <item row="1" column="3">
           <widget class="HeaderTableItem" name="dayThursday" native="true"/>
          </item>
          <item row="2" column="2">
           <widget class="HeaderTableItem" name="daySunday" native="true"/>
          </item>
          <item row="2" column="0">
           <widget class="HeaderTableItem" name="dayFriday" native="true"/>
          </item>
          <item row="1" column="1">
           <widget class="HeaderTableItem" name="dayTuesday" native="true"/>
          </item>
          <item row="0" column="0" colspan="4">
           <layout class="QHBoxLayout" name="layoutWeekChoose">
            <item>
             <widget class="QFrame" name="frameWeekNavigation">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="layoutWeekNavigation">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="spcWeekChooseLeft">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btnWeekPrevious">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Previous week</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/images/arrow_left.png</normaloff>:/images/arrow_left.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>34</width>
                   <height>34</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frameWeekChoose">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame {
background: white;
padding: 2px;
border-radius: 4px;
}

#frame {
border: 1px solid lightgray;
}

QLabel {
padding: 2px 1px;
}</string>
                 </property>
                 <layout class="QHBoxLayout" name="layoutWeekChoose_2">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="leftMargin">
                   <number>2</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>2</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="layoutChoosedYear">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="lblYear">
                      <property name="font">
                       <font>
                        <family>Roboto</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Year:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="lblChoosedYear">
                      <property name="font">
                       <font>
                        <family>Roboto</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="layoutChoosedMonth">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="lblMonth">
                      <property name="font">
                       <font>
                        <family>Roboto</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Month:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="lblChoosedMonth">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Roboto</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="layoutChoosedWeek">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>3</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="lblWeek">
                      <property name="font">
                       <font>
                        <family>Roboto</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Week:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="lblChoosedWeek">
                      <property name="minimumSize">
                       <size>
                        <width>30</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <family>Roboto</family>
                        <pointsize>12</pointsize>
                       </font>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnSelectWeek">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>34</width>
                      <height>34</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Roboto</family>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="toolTip">
                     <string>Choose week</string>
                    </property>
                    <property name="icon">
                     <iconset resource="resources.qrc">
                      <normaloff>:/images/arrow_down.png</normaloff>:/images/arrow_down.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>28</width>
                      <height>28</height>
                     </size>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnWeekNext">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Next week</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/images/arrow_right.png</normaloff>:/images/arrow_right.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>34</width>
                   <height>34</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="spcWeekChooseRight">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabAffairs">
         <attribute name="title">
          <string>Affairs</string>
         </attribute>
         <layout class="QGridLayout" name="layoutAffairs">
          <property name="spacing">
           <number>9</number>
          </property>
          <item row="0" column="1">
           <widget class="HeaderTableItem" name="tableEducationAffairs" native="true"/>
          </item>
          <item row="0" column="3">
           <widget class="HeaderTableItem" name="tableWorkAffairs" native="true"/>
          </item>
          <item row="1" column="1">
           <widget class="HeaderTableItem" name="tablePersonalAffairs" native="true"/>
          </item>
          <item row="1" column="3">
           <widget class="HeaderTableItem" name="tablePlaces" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabShaping">
         <attribute name="title">
          <string>Forming</string>
         </attribute>
         <layout class="QGridLayout" name="layoutForming">
          <property name="spacing">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="layoutFormVisitPlan">
            <item>
             <widget class="QLabel" name="lblFormVisitPlan">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Formation of a visit plan</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ImageButton" name="btnFormVisitPlan">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Form visit plan</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="layoutFormPersonalAffairs">
            <item>
             <widget class="QLabel" name="lblFormPersonalAffairs">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Formation of personal goals</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ImageButton" name="btnFormPersonalAffairs">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Form personal goals</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HeaderTableItem</class>
   <extends>QWidget</extends>
   <header>headertableitem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageButton</class>
   <extends>QPushButton</extends>
   <header>imagebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
